cmd/lookout/serve.go

```diff
@@ -46,8 +46,16 @@ type ServeCommand struct {
 	analyzers map[string]lookout.AnalyzerClient
 }
 
+// Config holds the main configuration
+type Config struct {
+	Analyzers []lookout.AnalyzerConfig
```

Comment:
Maybe we can do

```go
type Config struct {
   lookout.ServerConfig
   Providers struct {
      ...
   }
}
```

In this case, we don't need to change 2 structure when we add more configuration options in yaml file.

Reply:
That was my original intention, but couldn't make it work with yaml unmarshalling.

Turns out there is a way, but the documentation for it is hidden in [an example](https://godoc.org/gopkg.in/yaml.v2#Unmarshal) and not in the method docs! 

Fixed in ca5dfe7.